async-request-reply/deploy.bicep

```diff
@@ -6,7 +6,7 @@ var hostingPlanName = 'app-reqrep'
 var functionAppName = 'fapp-reqrep-${uniqueString(subscription().subscriptionId, resourceGroup().id)}'
 var serviceBusNamespaceName = toLower('sb-reqrep-${uniqueString(subscription().subscriptionId, resourceGroup().id)}')
 var deploymentStorageContainerName = 'app-package-${uniqueString(subscription().subscriptionId, resourceGroup().id)}'
-var appInsigthName = 'appinsigth-${uniqueString(subscription().subscriptionId, resourceGroup().id)}'
+var appInsightName = 'appinsigth-${uniqueString(subscription().subscriptionId, resourceGroup().id)}'
```

Comment:
"appinsigth" is still in the actual resource's name here.  Want to try to fix that as well?